docs: update plugin documentation (#888)
jackyzha0/quartz: docs/plugins/ComponentResources.md

---
title: ComponentResources
tags:
  - plugin/emitter
---

> This plugin manages and emits the static resources required for the Quartz framework. This includes CSS stylesheets and JavaScript scripts that enhance the functionality and aesthetics of the generated site.

Reviewer: ```suggestion
This plugin emits all static resources required to render the page like stylesheets and JavaScript required for Quartz to render your site. 
```

Reviewer 2: maybe mention the `cdnCaching` option under `configuration.theme` here as well

Reviewer: Something along the line of:

```prolog
`cdnCaching=false` will download the fonts to make the final build full self-container. However worth noting that it might be less efficient comparing to edge-caching for hosted cdn fonts.
```

Author: Added a note and link th the [[configuration]] page and extended the sentence there.
